src/Freya.Machine.Extensions.Http/Domain.fs

```diff
@@ -180,46 +138,16 @@ module ContentNegotiation =
                          | _ -> false
             <!> negotiated acceptCharset supported
 
-    (* Encoding *)
+    (* ContentCoding *)
 
     [<RequireQualifiedAccess>]
-    module Encoding =
```

Comment:
Oh, yes it should. I'd missed that people might have relied on that. I'll add an obsolete shim wrapping the moved logic in Arachne. Thanks for spotting that!